hash.c

```diff
@@ -1857,27 +1860,35 @@ rb_hash_initialize(int argc, VALUE *argv, VALUE hash)
  *  but the argument is not an array of 2-element arrays or a
  *  {Hash-convertible object}[doc/implicit_conversion_rdoc.html#label-Hash-Convertible+Objects]:
  *
- *    Hash[:foo] # Raises ArgumentError (odd number of arguments
- *    Hash[ [ [:foo, 0, 1] ] ] # Raises ArgumentError (invalid number of elements (3 for 1..2))
+ *    # Raises ArgumentError (odd number of arguments
+ *    Hash[:foo]
+ *    # Raises ArgumentError (invalid number of elements (3 for 1..2))
+ *    Hash[ [ [:foo, 0, 1] ] ]
  *
  *  Raises an exception if the argument count is odd and greater than 1:
  *
- *    Hash[0, 1, 2] # Raises ArgumentError (odd number of arguments for Hash)
+ *    # Raises ArgumentError (odd number of arguments for Hash)
+ *    Hash[0, 1, 2]
```

Comment:
This is almost perfectly my preferred style, but what I tend to do is always follow one of two conventions:

- If you are making a statement of fact, it's a sentence, finish it with a full stop. This only occurs when documenting methods - it should almost never happen in the body of code.

- If you are explaining or making some other assertion about the behaviour of the code which follows, finish with a colon. This almost always applies to statements in the body of a method, because they are actually executed.

If you follow this convention, I think you clearly signal that the comment applies the the following line.

Reply:
So, colon here, then?

Reply:
Colons added.